src/content/docs/tutorial-infrastructure-monitoring/investigate-spike-CPU.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Investigate a spike in load
+metaDescription: Solve system outages by investigating resource bottlenecks across your system infrastructure.
+---
+
+You can use the infrastructure hosts pages to troubleshoot different incidents within your system. In the last doc, we discussed the benefits of fully instrumenting your system with recommendations from the guided install, then walked through a general example of how and why you might filter down to a set of hosts. This doc builds off that skill and walks you through how you might troubleshoot a real-world example.  
```

Comment:
might do like:

the infrastructure **Hosts** UI page...
with the idea that the various tabs on that page are all part of the same UI page (which is at least how I worded it for docs)